src/main/java/com/paloaltonetworks/osc/api/PANManagerSecurityGroupApi.java

```diff
@@ -87,7 +92,7 @@ public void updateSecurityGroup(String sgId, String name, SecurityGroupMemberLis
 
     @Override
     public void deleteSecurityGroup(String sgId) throws Exception {
-        this.showOperations.deleteDAGTag(sgId);
+        this.client.deleteDAGTag(sgId);
```

Comment:
when you delete the security group we should delete all the ip addresses under the tag right?